templates/directives/autoreply.html

```diff
@@ -0,0 +1,11 @@
+<span ng-if="hasState">
+  <task-state state="task.state"></task-state>
+  &nbsp;
```

Comment:
This looks like a hack - why is this necessary? There are some below too...

Reply:
It's actually copied from a filter https://github.com/medic/medic-webapp/pull/3998/files#diff-59129c7e90017db0b53628a428717201L82. I've changed that now so it looks more human-friendly